sabelotodo/app.py

```diff
@@ -13,5 +15,11 @@
 def hello():
```

Comment:
This code and `all_items()` below are duplicated here and in `routes.py`. Let's figure out which version is actually used/tested and drop the other one.

Reply:
Good catch. I think we can now drop this file entirely.